### ISSUE TITLE
Support login with SSO

### ISSUE THREAD

**Issue by ThePlenkov** [lastpass/lastpass-cli#652]
Dear lastpass,

My company is using SSO and I don't think we have master passwords available.

Unfortunately because of that I'm not able to use this CLI because of this password popup.

It would be nice to have something like `lpass login --sso` which should redirect to the browser and allow to complete authentication there. I'm using a lot CLI's from different vendors which support that and it's very convenient.

Thanks!

**Comment by olly-reardon** [comment]
Using `lpass` in an enterprise environment which enforces sso would be very useful. I upvote this also.

For now - I just use https://www.passwordstore.org/

**Comment by tatsuo48** [comment]
I'm facing the same issue.

**Comment by robermp** [comment]
+1

**Comment by gabefighter99** [comment]
+1

**Comment by PwshKen** [comment]
Adding another vote for CLI to support SSO.

**Comment by bbraunstein** [comment]
+1 for SSO support. Would love to do away with ClickOps in the team and provide CLI support for automation.

**Comment by dithmer** [comment]
+1 here. CLI is great, but I would love to be able to use it with SSO

**Comment by ScottJMcMullan** [comment]
+1 this would be great for my workflow.

**Comment by sneakyvv** [comment]
+1 This isn't just useful for our workflow, but critical. Without it we can't easily share secrets via automated scripts.

**Comment by alessandroaussems** [comment]
Another one

**Comment by wannevancamp** [comment]
+1 for SSO support

**Comment by dani882** [comment]
I was looking for this solution as well, I really need for share secret using automation. So +1 for SSO support

**Comment by ravemir** [comment]
Sad to see confirmation that we can't do this right now. +1 for us (use case was precisely automating the decommissioning of Shared Secrets)

**Comment by lunarplasma** [comment]
> Sad to see confirmation that we can't do this right now. +1 for us (use case was precisely automating the decommissioning of Shared Secrets)

Yeah, we want to use the CLI for automation too. It would save a great deal of time, but can't do so because we're on SSO.

**Comment by a-abella** [comment]
One more here. I've been gushing to my employer about 1Password and Bitwarden both having CLIs with SSO support and first-class Kubernetes operators. With LastPass if I need to record 100 new API keys I end up making a Secure Note that says "go look at [some other secrets source]" 

2026 update: our company has left LastPass with this being one of the main factors.

**Comment by xTrevski** [comment]
+1 - Adding another vote for SSO

**Comment by intjonathan** [comment]
How is this still open?! How is SSO a supported feature if you can't use it in the only non-browser client? Embarrassing.

**Comment by neilharris1** [comment]
+1

**Comment by a1ex-var1amov** [comment]
+1; I am truly surprised it has no SSO support.

**Comment by DaniloGT** [comment]
+1 Up

**Comment by oliverkuchies** [comment]
+1 would love to see this for our organisation
